doc/QCDB.md

```diff
@@ -0,0 +1,198 @@
+
+QCDB
+---
+
+<!--TOC generated with https://github.com/ekalinin/github-markdown-toc-->
+<!--./gh-md-toc --insert --no-backup --hide-footer --indent 3 QualityControl/doc/Advanced.md -->
+<!--ts-->
+   * [QCDB vs CCDB](#qcdb-vs-ccdb)
+   * [Details on the data storage format in the QCDB](#details-on-the-data-storage-format-in-the-qcdb)
+   * [Custom metadata for QC objects in the QCDB](#custom-metadata-for-qc-objects-in-the-qcdb)
+   * [Instructions to move an object in the QCDB](#instructions-to-move-an-object-in-the-qcdb)
+   * [Accessing objects in CCDB](#accessing-objects-in-ccdb)
+   * [Access GRP objects with GRP Geom Helper](#access-grp-objects-with-grp-geom-helper)
+   * [Global Tracking Data Request helper](#global-tracking-data-request-helper)
+   * [Local CCDB setup](#local-ccdb-setup)
+<!--te-->
+
+## QCDB vs CCDB
+
+The MonitorObjects generated by Quality Control are stored in a dedicated repository called **QCDB**.
+The run conditions, on the other hand, are located in another, separate database, called **CCDB**.
+
+Both are based on a technology called _CCDB_ which does not help with the confusion...
+
+## Details on the data storage format in the QCDB
+
+Each MonitorObject is stored as a TFile in the QCDB.
+It is therefore possible to easily open it with ROOT when loaded with alienv. It also seamlessly supports class schema evolution.
+
+The MonitorObjects are stored at a path which is enforced by the qc framework : `/qc/<detector code>/MO/<task name>/object/name`
+Note that the name of the object can contain slashes (`/`) in order to build a sub-tree visible in the GUI.
+The detector name and the taskname are set in the config file :
+
+```json
+"tasks": {
+  "QcTask": {       <---------- task name
```

Comment:
technically it's task ID. task name would be at:
```
  "QcTask": {       <---------- task ID
Comment
    "active": "true",
    "taskName": "QcTask", <--------- task name
    "className": "o2::quality_control_modules::skeleton::SkeletonTask",
    ...
```
If task name is missing, task ID is used. The split was done to allow for the same task names for multiple detectors in the same config files.

It is correct that task name is used in the QCDB paths though.

I leave it up to you if you want to go into these technicalities, but it might be worth it, given that this distinction is very much used in async (e.g. https://github.com/AliceO2Group/O2DPG/blob/71b020ddb74e7feb4f485c790921c16650f1418d/DATA/production/qc-async/its.json#L28C30-L28C32 )